2.0/root/opt/app-root/etc/puma.cfg

```diff
@@ -9,6 +9,15 @@ def get_max_memory()
     (2 ** (8*0.size - 2) - 1)
 end
 
+def per_worker_memory()
+    bytes = ENV.fetch('MEMORY_BYTES_PER_WORKER', '0').to_i
+    if bytes == 0
+        # Unless told otherwise, allow 256MiB per worker
```

Comment:
Also: comment should probably be moved to method description(above the `def`).
Other wise it LGTM.